### ISSUE TITLE
The offline dir went to my home dir instead the present project dir after running "yarn config set yarn-offline-mirror ./myDirectory ", 

### ISSUE THREAD

**Issue by yuqingc** [yarnpkg/yarn#5128]
**Do you want to request a *feature* or report a *bug*?**
Report a bug (probably a bug)

**What is the current behavior?**
I was trying to use offline yarn. I cd to my project directory and run "yarn config set yarn-offline-mirror ./my-offline" and then move the .yarnrc file to the present project directory by running "mv ~/.yarnrc ./"
After that, I removed the node_module/ and yarn.lock
Then I ran "yarn add"
Finally, I find the "my-offline" folder in my home directory

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

The "my-offline" folder should be in my project directory

**Please mention your node.js, yarn and operating system version.**
OS version: Ubuntu 17.10
Node version: 8.9.3
Yarn version: 1.2.1

**Comment by yuqingc** [comment]
Help!!!

**Comment by jackblackCH** [comment]
Same here: I also expect that running `yarn config set yarn-offline-mirror ./my-offline` from my project directory would set the path in `.yarnrc` relative to my project directory.

**Comment by wchfish** [comment]
I have the same problem too.

**Comment by yuqingc** [comment]
@torifat

**Comment by BYK** [comment]
@arcanis I agree that yarn should modify or create the project-level rc file by default. This would be a breaking change however so sounds like this is a good candidate for 2.0?

**Comment by senseluo** [comment]
same here

**Comment by kohlerm** [comment]
Yarn changing this to an absolute build path, which in almost all cases will be some specific users home directory makes using the offline mode with cloud foundry impractical because the buildpack will not run with that same user. 
Also when I check-in the .yarnrc file this will break the project for all other developers.
